pkg/apis/deviceplugin/v1/fpgadeviceplugin_types.go

```diff
@@ -71,8 +71,11 @@ type FpgaDevicePluginStatus struct {
 // +kubebuilder:printcolumn:name="Ready",type=integer,JSONPath=`.status.numberReady`
 // +kubebuilder:printcolumn:name="Node Selector",type=string,JSONPath=`.spec.nodeSelector`
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
+// +operator-sdk:csv:customresourcedefinitions:displayName="FPGA Device Plugin"
```

Comment:
for all `displayName` I'd add Intel where applicable and write out the acronym (see QAT/SGX below)... or is there a character limit?

Reply:
I've just checked the preview page: even if there is a limit all the display names fit still it. Though the description fades away after "...device plugin responsible for...". Not sure it's a problem though.